### PR TITLE
Minor documentation improvement.

### PR DESCRIPTION
**Pull request by trifonnt**
Working on Ubuntu 20.04 version and I had to execute 2 commands in order to get everything working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 ## Usage
 * Clone or download to a folder.
 * If you want to install and run postgresql docker container image, make sure your current login user has the necessary permission to run docker (https://www.baeldung.com/linux/docker-run-without-sudo).
+* Run **autoload bashcompinit**
+* Run **bashcompinit**
```
Comment (hengsin) [inline README.md:13]: I think that's because you are not using bash shell (zsh perhaps ?)

Comment (trifonnt) [reply README.md:13]: Yes, you are right. Currently zsh is used.

Comment (hengsin) [reply README.md:13]: So the change above is not entirely correct, the 2 command is only needed if you are using zsh.

Comment (trifonnt) [reply README.md:13]: Yes, you are right. This commands are needed only if zsh is used.
I can put comments after this commands that they are needed only if zsh is used.

Comment (hengsin) [reply README.md:13]: please add the zsh condition, this is not needed for people using the bash shell.

File: README.md [hunk 2]
```diff
@@ -9,12 +9,17 @@
...
 * Run **source completion.bash** to turn on auto completion for bundle shell scripts.
 * Run **./setup.sh --help** to get help on options available.
 
 Examples of usage:
 
+    ./setup.sh --branch release-10 --repository-url git@github.com:trifonnt/idempiere.git --docker-postgres-create
```
Comment (hengsin) [inline README.md:19]: avoid use of hard coded value in example (trifonnt in this case) or you must accomplish that with documentation for the example.

File: README.md (hunk 1 above)
Comment (trifonnt) [reply README.md:13]: > please add the zsh condition, this is not needed for people using the bash shell.

As a comment after the command or as comment above the command?

File: README.md (hunk 2 above)
Comment (hengsin) [reply README.md:19]: ic, please remove it from the PR then.

File: README.md (hunk 1 above)
Comment (hengsin) [reply README.md:13]: should be something like, if you are using zsh instead of bash shell, run the following 2 command to enable bash completion.

Comment (trifonnt) [reply README.md:16]: @hengsin  Done. Please let me know if there is anything further which can be improved?

File: README.md (hunk 2 above)
Comment (trifonnt) [reply README.md]: @hengsin  Done. Used standard idempiere repository. Please let me know if there is anything further which can be improved?

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**